libnd4j/tests_cpu/layers_tests/DeclarableOpsTests2.cpp

```diff
@@ -4177,3 +4177,62 @@ TEST_F(DeclarableOpsTests2, lstmCell_test12) {
 
     
 }
+
+
+TEST_F(DeclarableOpsTests2, ctc_loss_test1) {
+    constexpr int FRAME_LEN = 6 ;
```

Comment:
Mind commenting on where you get the test case from so it's easily reproducible outside of dl4j?

Reply:
I generated a random one with the help of python

Reply:
added gist link of the test